### ISSUE TITLE
Error in workflow

### ISSUE THREAD

**Issue by amarmadhavan** [deanishe/alfred-vpn-manager#12]
Getting a strange error and I'm not sure what to check here.
The following is the output I'm getting from the debug log for the workflow. Any advice would be appreciated!
```
[2018-07-16 23:40:57][input.scriptfilter] Queuing argument ''
[2018-07-16 23:40:57][input.scriptfilter] Script with argument '' finished
[2018-07-16 23:40:57][ERROR: input.scriptfilter] Code 1: .
23:40:57 workflow.py:2055 DEBUG    ---------- VPN Manager (3.0.2) ----------
23:40:57 workflow.py:1462 DEBUG    reading settings from /Users/USER/Library/Application Support/Alfred 3/Workflow Data/net.deanishe.alfred.vpn.manager/settings.json
23:40:57 workflow.py:2339 DEBUG    update check not due
23:40:57 vpn.py:523 DEBUG    args : {'--all': False,
 '--help': False,
 '<name>': None,
 '<query>': u'',
 'conf': False,
 'connect': False,
 'disconnect': False,
 'list': True}
23:40:57 workflow.py:2074 ERROR    Command '['mdls', '-raw', '-name', 'kMDItemCFBundleIdentifier', '/Applications/Tunnelblick.app\n/Library/StagedExtensions/Applications/Tunnelblick.app']' returned non-zero exit status 1
Traceback (most recent call last):
  File "/Users/USER/Library/Application Support/Alfred 3/Alfred.alfredpreferences/workflows/user.workflow.B13B074F-5DEE-44AB-BEAF-5BB14CF5D519/workflow/workflow.py", line 2067, in run
    func(self)
  File "./vpn.py", line 526, in main
    return do_list(args.get('<query>'))
  File "./vpn.py", line 418, in do_list
    app = get_app()
  File "./vpn.py", line 275, in get_app
    if not app.installed:
  File "./vpn.py", line 126, in installed
    return self.info is not None
  File "./vpn.py", line 119, in info
    self._info = appinfo(self.name)
  File "/Users/USER/Library/Application Support/Alfred 3/Alfred.alfredpreferences/workflows/user.workflow.B13B074F-5DEE-44AB-BEAF-5BB14CF5D519/workflow/util.py", line 251, in appinfo
    bid = run_command(cmd).strip()
  File "/Users/USER/Library/Application Support/Alfred 3/Alfred.alfredpreferences/workflows/user.workflow.B13B074F-5DEE-44AB-BEAF-5BB14CF5D519/workflow/util.py", line 150, in run_command
    return subprocess.check_output(cmd, **kwargs)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/subprocess.py", line 573, in check_output
    raise CalledProcessError(retcode, cmd, output=output)
CalledProcessError: Command '['mdls', '-raw', '-name', 'kMDItemCFBundleIdentifier', '/Applications/Tunnelblick.app\n/Library/StagedExtensions/Applications/Tunnelblick.app']' returned non-zero exit status 1
23:40:57 workflow.py:2097 DEBUG    ---------- finished in 0.086s ----------
[2018-07-16 23:40:57][input.scriptfilter] {"items": [{"valid": false, "subtitle": "Command '['mdls', '-raw', '-name', 'kMDItemCFBundleIdentifier', '/Applications/Tunnelblick.app\\n/Library/StagedExtensions/Applications/Tunnelblick.app']' returned non-zero exit status 1", "icon": {"path": "/System/Library/CoreServices/CoreTypes.bundle/Contents/Resources/AlertStopIcon.icns"}, "title": "Error in workflow 'VPN Manager'"}]}
```

**Comment by deanishe** [comment]
Okay. It seems to be finding multiple results when it searches for Tunnelblick.

I'll make the search more discriminating. Let me crack open a beer and have a look…

**Comment by deanishe** [comment]
Should be fixed in [v3.0.3](https://github.com/deanishe/alfred-vpn-manager/releases/tag/v3.0.3)
